modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/yarn.lock

```diff
@@ -1,11643 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think that we wrong deleted this file, probably need to restore it.

Reply:
Hey @igordmf We got some problems while rebasing with this file. But deleting it will cause no harm. Just run yarn install again

Reply:
Ok man!!